src/web/frontend/redux/actions/fetchData.tsx

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
```

Comment:
you've got everything in a single file
separate it into several files:
one should contain actions themselves,
the second should not be located in actions folder at all, but contain logic for fetching the data with axios